lighthouse-core/test/computed/page-dependency-graph-test.js

```diff
@@ -9,8 +9,8 @@ const PageDependencyGraph = require('../../computed/page-dependency-graph.js');
 const BaseNode = require('../../lib/dependency-graph/base-node.js');
 const NetworkRequest = require('../../lib/network-request.js');
 
-const sampleTrace = require('../fixtures/traces/progressive-app-m60.json');
-const sampleDevtoolsLog = require('../fixtures/traces/progressive-app-m60.devtools.log.json');
+const sampleTrace = require('../fixtures/traces/iframe-m79.trace.json');
+const sampleDevtoolsLog = require('../fixtures/traces/iframe-m79.devtoolslog.json');
```

Comment:
The M60 trace has the old [`TracingStartedInPage`](https://github.com/GoogleChrome/lighthouse/blob/ae729f63a74ad9d4a16bd8ba8a88684891eea0b8/lighthouse-core/lib/tracehouse/trace-processor.js#L446-L449) which uses uses a memory address as the frame ID instead of the same ID in the DT log.

Reply:
I wonder if we should refresh many of these old traces. m60 is from 2017-07

Reply:
Not a bad idea, but wouldn't help this PR anymore.

After piping `artifacts.URL` everywhere, this change isn't really necessary.